helpers/providers/huggingface.ts

```diff
@@ -0,0 +1,61 @@
+import prompts from "prompts";
+import { ModelConfigParams } from ".";
+import { questionHandlers, toChoice } from "../../questions/utils";
+
+const MODELS = ["HuggingFaceH4/zephyr-7b-alpha"];
+type ModelData = {
+  dimensions: number;
+};
+const EMBEDDING_MODELS: Record<string, ModelData> = {
+  "all-MiniLM-L6-v2": { dimensions: 384 },
+};
```

Comment:
@tomaarsen please add other models that are running well locally. (I added a model selector that you can call with `create-llama --ask-models`)